### PR TITLE
Use exponential back-off for DescribeCluster

### PR DESCRIPTION
**Pull request by vitillo**
See
https://aws.amazon.com/premiumsupport/knowledge-center/emr-cluster-status-throttling-error/

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-airflow/49)

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

File: dags/operators/emr_spark_operator.py [hunk 1]
```diff
@@ -139,7 +140,11 @@ def execute(self, context):
                                                                                                                                           self.job_flow_id))
 
         while True:
-            result = client.describe_cluster(ClusterId = self.job_flow_id)
+            @retry(wait_exponential_multiplier=1000, wait_exponential_max=300000, stop_max_delay=1800000)
```
Comment (maurodoglio) [inline dags/operators/emr_spark_operator.py]: Can you please add a comment for what this means in practice? Like
```python
# Start with 1s, increase up to 300s, Stop after 30m
```

**Comment by vitillo** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[dags/operators/emr_spark_operator.py, line 143 at r1](https://reviewable.io:443/reviews/mozilla/telemetry-airflow/49#-KSqFHoPfLTU6_9fcFbn-r1-143:-KSqRXGz5vl8FQm0BrJr:b-896fix) ([raw file](https://github.com/mozilla/telemetry-airflow/blob/680ce5412a9e500bbfb07203a8bc008f7574724a/dags/operators/emr_spark_operator.py#L143)):*
<details><summary><i>Previously, maurodoglio (Mauro Doglio) wrote…</i></summary>
> Can you please add a comment for what this means in practice? Like
> 
> ```python
> 
> # Start with 1s, increase up to 300s, Stop after 30m
> 
> ```
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/telemetry-airflow/49)*

**Comment by maurodoglio** [comment]
:lgtm:

---

Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/telemetry-airflow/49#-:-KSr4VqQpFz6iVd91GqO:bnfp4nl)*

**Pull request merged**